marimo/_runtime/runner/hooks.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Marimo. All rights reserved.
+__all__ = [
+    "PREPARATION_HOOKS",
+    "PRE_EXECUTION_HOOKS",
+    "POST_EXECUTION_HOOKS",
+    "ON_FINISH_HOOKS",
```

Comment:
maybe worth a comment - whats the difference between `post-execution` and `on-finish`?